plugins/spam.py

```diff
@@ -0,0 +1,44 @@
+from itertools import chain
+
+from errbot import BotPlugin
+
+from plugins import constants
+
+
+class SpammingAlert(BotPlugin):
+    """
+    A callback to alert the user that they might be spamming.
```

Comment:
s/callback/plugin/

The callback is the `callback_message` method below.

You could say

> A plugin which uses a callback to alert...

But line length might be a problem with that.

Reply:
I've made changes, but it still says changes are requested, did I miss something?